client/gutenberg/extensions/presets/jetpack/editor-shared/get-site-fragment.js

```diff
@@ -26,5 +26,14 @@ export default function getSiteFragment() {
 		return siteFragment || null;
 	}
 
+	// Gutenberg in Jetpack adds a site fragment in the initial state
+	if (
+		window &&
```

Comment:
This could likely look a bit better if we use `lodash.get()`, but I don't have strong preferences.

Reply:
Thanks. I've used it now to consolidate this and the `return` afterwards.